app/controllers/gardens_controller.rb

```diff
@@ -0,0 +1,11 @@
+class GardensController < ApplicationController
+
+  def index
+    @gardens = Garden.all
+  end
+
+  def show
+    @garden = Garden.find(params[:id])
+  end
+
```

Comment:
@georgkloeck this line can be removed, otherwise, looks good!

Reply:
okay, great